server/controllers/stock/requisition/requisition.js

```diff
@@ -155,7 +199,85 @@ exports.update = async (req, res, next) => {
       delete requisition.uuid;
     }
 
-    requisition.date = requisition.date ? new Date(requisition.date) : new Date();
+    if (requisition.movementRequisition) {
+      const dataMovementRequisition = db.convert(
+        req.body.movementRequisition, ['stock_requisition_uuid', 'document_uuid'],
+      );
+
+      delete requisition.movementRequisition;
+
+      // Just to make a link between the stock issues coming from the requisition
+      await db.exec('INSERT INTO stock_requisition_movement SET ?;', dataMovementRequisition);
+
+      const checkRequisitionBalance = `
+      SELECT COUNT(balance.inventory_uuid) AS numberInventoryPartial
+        FROM (
+        SELECT req.inventory_uuid, req.code, req.inventoryType,
+        (req.quantity - IF(mouv.quantity, mouv.quantity, 0)) AS quantity
+            FROM (
+              SELECT BUID(i.uuid) inventory_uuid, i.code, it.text as inventoryType, sri.quantity
+              FROM stock_requisition_item sri
+              JOIN inventory i ON i.uuid = sri.inventory_uuid
+              JOIN inventory_type it ON i.type_id = it.id
```

Comment:
Same comment about the inventory_type here.  It doesn't look like you need it.

Reply:
:+1: